loader/stage0/abl/stage0.c

```diff
@@ -161,6 +161,7 @@ void stage0_main(
 		print_panic("vmm mem size is smaller than %u\n", MINIMAL_EVMM_RT_SIZE);
 		goto fail;
 	}
+	barrier_size = barrier_size/2;
```

Comment:
I know your point now. I suggest to ignore abl. we will delete abl stage0 later.